### DISCUSSION TITLE
Migration from V2 to V3 - 404 Page Bug (Could it be something in settings?)

### DISCUSSION THREAD

**Discussion by Omerko96** [statamic/cms#5162] [Troubleshooting]
Hello, I have a few questions related to migration, please, if someone had the same or similar issues, share your experience here, Thanks!

I migrated my website from **V2 to V3**, using this guide **--->> https://github.com/statamic/migrator** Simple guide in 9 steps and it worked.

However, when I run my newly migrated website, I cannot get any pages, and also, I cannot see any assets on my screen. I double-checked and everything is migrated, but somehow **I am getting a 404 page**, which is the **default 404 page from Statamic Starter Project**.

Also, I can see my partials, like my navigation bar and my footer, and in between of those is my 404 page rendered, which means that partials are working, while page files are not, so it can be something in settings.

Also, I cannot see my assets on my screen, for example, inside of my navigation bar, I do have a logo, which is not presented at all.

**Did anyone have a similar issue? Also, did anyone solve this issue before?**

If any of you people from Statamic see this post, please ask for any additional information, I will surely give those to you as I wish to solve this.

Thanks a lot!

**Comment by edalzell** [comment]
Let's see your `sites.php`

**Comment by duncanmcclean** [comment]
Set the APP_URL in your .env to your local URL.
